src/components/editions/byline.tsx

```diff
@@ -18,26 +19,28 @@ import type { FC, ReactNode } from 'react';
 import { getThemeStyles } from 'themeStyles';
 import EditionsAvatar from './avatar';
 import { ShareIcon } from './shareIcon';
-import {
-	articleWidthStyles,
-	tabletArticleMargin,
-	wideArticleMargin,
-} from './styles';
+import { borderWidthStyles } from './styles';
 
 // ----- Styles ----- //
 
 const interviewStyles = css`
-	margin-left: ${remSpace[2]};
-	margin-right: ${remSpace[2]};
+	padding-left: ${remSpace[2]};
+	padding-right: ${remSpace[2]};
 
 	${from.tablet} {
-		margin-left: ${tabletArticleMargin}px;
+		box-sizing: border-box;
+		margin-left: ${remSpace[6]};
+		padding-left: ${remSpace[3]};
+		padding-right: ${remSpace[3]};
 	}
 
 	${from.wide} {
-		margin-left: ${wideArticleMargin}px;
+		margin-left: ${remSpace[24]};
+		left: ${remSpace[12]};
```

Comment:
This looks a bit suspect -  can we not just use the margin-left at `9rem` or even the value from the `styles` files that we've used elsewhere?

Reply:
Think this was just to favour remSpace but agreed, looks gnarly. Will revert this!